.evg.yml

```diff
@@ -388,7 +394,7 @@ functions:
           ./resources/run_adf.sh start &&
           ./gradlew -Dorg.gradle.java.home=${JAVA_HOME} runDataLoader &&
           ./gradlew -Dorg.gradle.java.home=${JAVA_HOME} clean integrationTest \
-              -x test  > gradle_output.log 2>&1 &
+              -x test --tests ADFIntegrationTest > gradle_output.log 2>&1 &
```

Comment:
This filter ensures we only run the `ADFIntegrationTest.java` tests for this function. The other, newly added function, `"run dc integration test"`, uses the filter `--tests DCIntegrationTest`. I thought this was a cleaner solution than adding a whole new package to the codebase and a new gradle task.